### PR TITLE
Close-to-final updates for DR8.

### PR DESCRIPTION
**Pull request by geordie666**
A few additional updates to deal with issues in #91 and clarifications raised as part of #92. 

The last remaining update for DR8 might be to the documentation indicating how to [access raw data through the NOAO Data Lab](http://www.legacysurvey.org/dr8/files/#raw-data). Otherwise, DR8 documentation should be complete for the foreseeable future.

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: At some point, could you check these final updates to the website, merge the PR and serve the new (hopefully final) version of the website? You may want to wait until @stephjuneau reaffirms the documentation indicating how to [access raw data through the NOAO Data Lab](http://www.legacysurvey.org/dr8/files/#raw-data), but you should be better placed than I to know more about that.

**Comment by weaverba137** [comment]
I will take a look at this today.

**Comment by weaverba137** [comment]
@stephjuneau, could you give an estimate of when the raw data access instructions might be updated?

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # legacysurvey Change Log
 
+
+## 8.0.1 (DR7, 2018-07-26)
```
Comment (weaverba137) [inline CHANGES.md]: Cut & paste from DR7

Comment (geordie666) [reply CHANGES.md]: Thanks for looking at this @weaverba137. Let's update the parenthetical and resolve this conversation whenever @stephjuneau replies and the PR gets merged. I don't want to change this today and then have the PR be merged into master next week...

Comment (weaverba137) [reply CHANGES.md]: That's fine.

**Comment by stephjuneau** [comment]
Sorry for being late to comment here. So the main difference from dr7 is that I thought we would serve the two ("north" and "south") footprints separately, which means that there would be two collections: "Legacy Survey: DR8-north" and "Legacy Survey: DR8-south", which I think is the most similar to how the files are served at NERSC? Unless you have an objection and want us to combine into only one collection? A closer examination also revealed another discrepancy, as written in the attached screenshot. I will follow with Sean McManus, who said he expects to work on this during this week.
LS_noao_screenshot

File: CHANGES.md (hunk 1 above)
Comment (stephjuneau) [reply CHANGES.md]: Hi @geordie666 & @weaverba137, to follow up from my reply below, please let me know if you have an opinion about serving the separate dr8-north and dr8-south collections, just the combined dr8 collection, or a third option which would be to serve 3 collections with the separated footprints, and the combined one.

> Sorry for being late to comment here. So the main difference from dr7 is that I thought we would serve the two ("north" and "south") footprints separately, which means that there would be two collections: "Legacy Survey: DR8-north" and "Legacy Survey: DR8-south", which I think is the most similar to how the files are served at NERSC? Unless you have an objection and want us to combine into only one collection? A closer examination also revealed another discrepancy, as written in the attached screenshot. I will follow with Sean McManus, who said he expects to work on this during this week.
> LS_noao_screenshot

**Comment by weaverba137** [comment]
This is how the files are organized at NERSC:  https://portal.nersc.gov/project/cosmo/data/legacysurvey/dr8/.  We will duplicate this layout *exactly* at NOAO.  However, this does not include the *raw* data files, and that is what we are trying to figure out here.

**Comment by stephjuneau** [comment]
OK, so then the next questions are whether to add "raw/" and "calibrated/" folders (I don't see the calibrated one either) and if so, I suppose they should go under the north/ and south/ subfolders so the resulting list would have:
- ftp://archive.noao.edu/public/hlsp/ls/dr8/north/raw/
- ftp://archive.noao.edu/public/hlsp/ls/dr8/south/raw/
- ftp://archive.noao.edu/public/hlsp/ls/dr8/north/calibrated/
- ftp://archive.noao.edu/public/hlsp/ls/dr8/south/calibrated/
- ftp://archive.noao.edu/public/hlsp/ls/dr8/north/coadd/
- ftp://archive.noao.edu/public/hlsp/ls/dr8/south/coadd/
Does that sound like a good solution?

**Comment by weaverba137** [comment]
That sounds fine to me.

**Comment by weaverba137** [comment]
@geordie666, have you incorporated @stephjuneau's suggestions?  We can't install right now because of NERSC being down, but I'd like to at least be able to do so early next week.

**Comment by geordie666** [comment]
@weaverba137: I haven't incorporated @stephjuneau's suggestions, yet, as I wasn't sure that everything was finalized on the Data Lab side. If @stephjuneau is certain that things will definitely be set up as she suggested in her previous comments, then I'll go ahead and add the information. @stephjuneau ?

**Comment by weaverba137** [comment]
Ah, yes, I suppose we ought to have the links functional.

**Comment by geordie666** [comment]
@weaverba137: Maybe we should merge this and serve it (if you have time)? Then we'll wait another few weeks for additional updates (first also merging #96, which fixed the cutout server links).

Note that the CHANGES.md log needs updated with the correct date, and [the final commit in my branch](https://github.com/legacysurvey/legacysurvey/tree/ADM-final-DR8), which seems to have become divorced from this PR for reasons that I don't understand. But hopefully you can merge things smoothly.

**Comment by weaverba137** [comment]
@geordie666 , are you saying you want to merge #96 *first*, then merge this?

I can clean up the changes file as needed.

**Comment by geordie666** [comment]
No, I'd merge this (#95) first and then #96. I know @dstndstn is working on a PR about the bit-issue in the source counts, so then we'd merge that (which is likely to have conflicts given that #95 also updated the source counts).

**Comment by weaverba137** [comment]
OK, merging now.

**Pull request merged**